app/controllers/webhook_deliveries_controller.rb

```diff
@@ -0,0 +1,29 @@
+class WebhookDeliveriesController < ApplicationController
+  respond_to :html, :js
+
+  before_action :set_namespace
+  before_action :set_webhook
+  before_action :set_webhook_delivery
```

Comment:
Since you are only using these helpers in the `update` method, and they are trivial, I'd prefer to move the code into the `update` method altogether.